### PR TITLE
Use SKV Partial Update in TPC-C Benchmark

### PR DESCRIPTION
**Pull request by YazhiFeng**

### CODE REVIEW

File: src/k2/cmd/tpcc/schema.h [hunk 1]
```diff
@@ -56,6 +56,32 @@ seastar::future<k2::WriteResult> writeRow(ValueType& row, k2::K2TxnHandle& txn)
...
+            K2DEBUG("partialUpdateRow failed: " << result.status);
+            return seastar::make_exception_future<k2::PartialUpdateResult>(std::runtime_error("partialUpdateRow failed!"));
+        }
+
+        return seastar::make_ready_future<k2::PartialUpdateResult>(std::move(result));
+    });
+}
+
+template<typename ValueType>
+seastar::future<k2::PartialUpdateResult>
```
Comment (jerryhfeng) [inline src/k2/cmd/tpcc/schema.h:73]: Suggestion: Could this fn combined into above one with input param vector element type parameterized?

Comment (jfunston) [reply src/k2/cmd/tpcc/schema.h:73]: yeah, you can add a template parameter for the fieldsToUpdate type, and then you only need one function.

File: src/k2/cmd/tpcc/transactions.h [hunk 2]
```diff
@@ -279,10 +284,13 @@ class NewOrderT : public TPCCTxn
                     *(line.Amount) = *(item.Price) * *(line.Quantity);
                     _total_amount += *(line.Amount);
                     line.DistInfo = stock.getDistInfo(*(line.DistrictID));
-                    updateStockRow(stock, line);
```
Comment (jfunston) [inline src/k2/cmd/tpcc/transactions.h:282]: You should delete the old definition of this function too.

**Review by jfunston — CHANGES_REQUESTED** [review]

**Comment by YazhiFeng** [comment]
Add a template parameter for partialUpdateRow and remove old func (updateStockRow).

**Review by jfunston — APPROVED** [review]

**Pull request merged**